docs/user/02-configuration.md

```diff
@@ -8,7 +8,9 @@ The CustomResourceDefinition (CRD) `eventings.operator.kyma-project.io` describe
 
 View the complete [Eventing CRD](https://github.com/kyma-project/eventing-manager/blob/main/config/crd/bases/operator.kyma-project.io_eventings.yaml) including detailed descriptions for each field.
 
-The CRD is equipped with validation rules and defaulting, so the CR is automatically filled with sensible defaults. You can override the defaults. The validation rules provide guidance when you edit the CR.
+The CRD is equipped with validation rules and defaulting, so the CR is automatically filled with sensible defaults. You can override the defaults. However, its backend is not set, it gets warning state and the Eventing Manager does not create any resources.
```

Comment:
```suggestion
The CRD is equipped with validation rules and defaulting, so the CR is automatically filled with default values. 

You can override the defaults. However, you must make sure that a backend is set; otherwise the Eventing Manager does not create any resources and goes into warning state.
```